src/Tabs/TabGroup.js

```diff
@@ -0,0 +1,108 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { TabContent } from './_TabContent';
+import React, { Component } from 'react';
+
+export class TabGroup extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedId: props.selectedId
+        };
+    }
+
+    static getDerivedStateFromProps(props, state) {
+        const prevProps = state.prevProps || {};
+        // Compare the incoming prop to previous prop
+        const selectedId =
+            prevProps.selectedId !== props.selectedId
+                ? props.selectedId
+                : state.selectedId;
+        return {
+            // Store the previous props in state
+            prevProps: props,
+            selectedId
+        };
+    }
+
+    // set selected tab
+    handleTabSelection = (event, id) => {
+        event.preventDefault();
+        this.setState({
+            selectedId: id
+        });
+
+        this.props.onTabClick(event, id);
+    };
+
+    // clone Tab element
+    cloneElement = (child) => {
+        return (React.cloneElement(child, {
+            onClick: this.handleTabSelection,
+            selected: this.state.selectedId === child.props.id
+        }));
+    }
+
+    // create tab list
+    renderTabs = () => {
+        return React.Children.map(this.props.children, (child) => {
+            return this.cloneElement(child);
+        });
+    };
+
+    // create content to show below tab list
+    renderContent = () => {
+        return React.Children.map(this.props.children, (child) => {
+            return (
+                <TabContent
+                    {...child.props.tabContentProps}
+                    selected={this.state.selectedId === child.props.id}>
+                    {child.props.children}
+                </TabContent>);
+        });
+    };
+
+    render() {
+        const {
+            children,
+            className,
+            selectedId,
+            tabGroupProps,
+            onTabClick,
+            ...rest } = this.props;
+
+        // css classes to use for tab group
+        const tabGroupClasses = classnames(
+            'fd-tabs',
+            className
+        );
+        return (
+            <React.Fragment>
+                <ul {...rest} className={tabGroupClasses}
+                    role='tablist'>
+                    {this.renderTabs(children)}
+                </ul>
+                {this.renderContent(children)}
+            </React.Fragment>
+        );
+    }
+}
+TabGroup.displayName = 'TabGroup';
+
+TabGroup.defaultProps = {
+    selectedId: '1',
```

Comment:
I think this prop's type needs to change to a number and represent the index of the selected tab.  The reason I say that is because using `id` attribute values requires the use of the `id` prop in the `Tab` component and the `selectedId` prop in `TabGroup`, both of which are currently optional.  It would be difficult to set a default `selectedId` since the tab's `id` value is not easily known.

The index of each tab could be bound to the `onClick` function being passed (in `cloneElement `) so there would be no other work needed.

Reply:
done.